get_mdtest_results.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+if [ $# -eq 0 ] ; then
+    echo "Usage: $0 <full path to results directory>"
+    exit 1
+fi
+
+RES_DIR=$1
+
+if [[ ! -d "$RES_DIR" ]] ;  then
+    echo "Result directory $RES_DIR does not exist"
+    exit 1
+fi
+
+cd $RES_DIR
+
+#Test name
+current=${PWD##*/}
+
+#Output files - raw* for extracting the results from tests and result* for final results in csv format into a file ready for plotting
+raw="./raw_$current"
+result="./result_$current"
+
+process_result() {
+    infile=$1
+    outfile=$2
+    if [ -f ./tmp ] ; then rm ./tmp ; fi
```

Comment:
instead of using hardcoded ./tmp file it is better to use mktemp command e.g.

> tmpfile = ```mktemp```

same comment for all other files where tmp is used

Reply:
Sure Will have this in the next version for review.